src/Components/Popover/Composed/ReflessPopover.tsx

```diff
@@ -12,49 +13,45 @@ interface Props extends Omit<PopoverProps, 'triggerRef'> {
   triggerStyle?: CSSProperties
 }
 
-export class ReflessPopover extends PureComponent<Props> {
-  public static readonly displayName = 'ReflessPopover'
+export type ReflessPopoverRef = HTMLSpanElement
 
-  public static defaultProps = {
-    ...PopoverDefaultProps,
-    testID: 'refless-popover',
-  }
-
-  private triggerRef = createRef<HTMLDivElement>()
-
-  public render() {
-    const {
+export const ReflessPopover = forwardRef<ReflessPopoverRef, Props>(
+  (
+    {
+      id,
+      type,
+      color,
+      style,
+      visible,
+      contents,
+      disabled,
+      position,
+      children,
+      hideEvent,
+      showEvent,
+      caretSize,
+      triggerStyle,
       distanceFromTrigger,
       enableDefaultStyles,
-      triggerStyle,
-      caretSize,
-      showEvent,
-      hideEvent,
-      children,
-      position,
-      disabled,
-      contents,
-      visible,
-      testID,
-      style,
-      color,
-      type,
-      id,
-    } = this.props
+      testID = 'refless-popover',
+    },
+    ref
+  ) => {
+    const triggerRef = createRef<HTMLDivElement>()
 
     return (
-      <>
+      <span ref={ref}>
```

Comment:
Wrapping this in another element might make it difficult to control layout. Presents an interesting case in which the component makes heavy use of `ref` already. Though it would be going against the grain, I don't think this component needs a `ref`. Hence the name, it is for using `Popover` without a `ref`